File: Medium/FirstAndLastIndexOfElementInSirtedArray.java
import java.util.ArrayList;
import java.util.List;
import java.util.stream.Collectors;
import java.util.stream.IntStream;

// https://leetcode.com/problems/find-first-and-last-position-of-element-in-sorted-array/


public class FirstAndLastIndexOfElementInSirtedArray {
    public static int[] elementRange(int[] nums, int target){
        int[] result = {-1, -1};
        List<Integer> array = new ArrayList<Integer>();
        array = IntStream.of(nums) // Create an IntStream from the array
        .boxed()  // Box primitive `int` to `Integer`
        .collect(Collectors.toList());

        if (array.size() == 0) return result;
        else {
            result[0] = array.indexOf(target);
            result[1] = array.lastIndexOf(target);
        }
        return result;
    }
}

File: Medium/FirstAndLastIndexOfElementInSirtedArray.py
# https://leetcode.com/problems/find-first-and-last-position-of-element-in-sorted-array/

class Solution(object):
    def searchRange(self, nums: list[int], target: int) -> list[int]:
        result = [-1, -1]
        if len(nums) == 0: return result
        elif len(nums) == 1 and nums[0] == target:
            result[0] = 0
            result[1] = 0
            return result
        elif len(nums) == 2 and nums[0] == target and nums[1] == target:
            result[0] = 0
            result[1] = 1
            return result
        else:
            result[0] = nums.index(target)
            result[1] - nums[::-1].index(target)
                        
        return result
    
a = Solution()
print(a.searchRange([2,2], 2))